wandb/sdk/wandb_run.py

```diff
@@ -3123,38 +3123,81 @@ def log_model(
                     - `/local/directory`
                     - `/local/directory/file.txt`
                     - `s3://bucket/path`
-            model_name: (str, optional) An artifact name. String containing only the following alphanumeric characters: dashes, underscores, and dots.
+            name: (str, optional) An artifact name. String containing only the following alphanumeric characters: dashes, underscores, and dots.
                 This will default to the basename of the path prepended with the current
                 run id  if not specified.
             aliases: (list, optional) Aliases to apply to this artifact,
                     defaults to `["latest"]`
 
+        Examples:
+            ```python
+            run.log_model(
+                path="/local/directory",
+                name="my_model_artifact",
+                aliases=["production"],
+            )
+            ```
+
+            Invalid usage
+            ```python
+            run.log_model(
+                path="/local/directory",
+                name="my_entity/my_project/my_model_artifact",
+                aliases=["production"],
+            )
+            ```
+
+        Raises:
+            ValueError: if name has invalid special characters
+
         Returns:
             None
         """
         self._log_artifact(
-            artifact_or_path=path, name=model_name, type="model", aliases=aliases
+            artifact_or_path=path, name=name, type="model", aliases=aliases
         )
 
     @_run_decorator._noop_on_finish()
     @_run_decorator._attach
-    def use_model(self, model_name: str) -> FilePathStr:
+    def use_model(self, name: str) -> FilePathStr:
         """Download a logged model artifact.
 
         Arguments:
-            model_name: (str) A model artifact name.
+            name: (str) A model artifact name.
```

Comment:
```suggestion
            name: (str) A model artifact name. This name must match a name of an existing model logged with the log_model method. 
```

Reply:
`model artifact 'name' must match a name of an existing model logged with the log_model method.`

This is untrue. It doesn't have to have been logged by the `log_model` method. I'll update this text accordingly.